readthedocs/doc_builder/config.py

```diff
@@ -31,7 +31,10 @@ def load_yaml_config(version):
     python_version = 3 if project.python_interpreter == 'python3' else 2
     allow_v2 = project.has_feature(Feature.ALLOW_V2_CONFIG_FILE)
     try:
-        sphinx_configuration = version.get_conf_py_path()
+        sphinx_configuration = path.join(
+            version.get_conf_py_path(),
+            'conf.py'
+        )
```

Comment:
It's not clear to me why this change is needed. Shouldn't the full path have already been returned by `version.get_conf_py_path()`?

Reply:
I was bitten by the name here too, this only returns the directory name. There is a test unskipped that covers this case.

Reply:
Also currently we are reading this from the database, that's why the builds don't fail